package.json

```diff
@@ -3999,6 +3999,446 @@
 											}
 										]
 									},
+									{
+										"key": "languageId:julia",
+										"name": "Julia",
+										"type": "bindings",
+										"bindings": [
+											{
+												"key": "x",
+												"name": "Execute code in REPL (ctrl+Enter)",
+												"icon": "play",
+												"type": "command",
+												"command": "language-julia.executeJuliaCodeInREPL",
+											},
+											{
+												"key": "d",
+												"name": "Show documentation",
+												"icon": "book",
+												"type": "command",
+												"command": "language-julia.show-documentation"
+											},
+											{
+												"key": "p",
+												"name": "Show plots",
+												"icon": "pulse",
+												"type": "command",
+												"command": "language-julia.show-plotpane",
+											},
+											{
+												"key": "w",
+												"name": "Focus on workspace view",
+												"icon": "play",
+												"type": "command",
+												"command": "REPLVariables.focus",
+											},
+											{
+												"key": "e",
+												"name": "+Environment/package",
+												"icon": "library",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "a",
+														"name": "Activate this environment",
+														"icon": "check",
+														"type": "command",
+														"command": "language-julia.changeCurrentEnvironment"
+													},
+													{
+														"key": "c",
+														"name": "Change current environment",
+														"icon": "issue-reopened",
+														"type": "command",
+														"command": "language-julia.changeCurrentEnvironment"
+													},
+													{
+														"key": "p",
+														"name": "Activate parent environment",
+														"icon": "root-folder-opened",
+														"type": "command",
+														"command": "language-julia.changeCurrentEnvironment"
+													},
+													{
+														"key": "m",
+														"name": "Choose module",
+														"icon": "symbol-variable",
+														"type": "command",
+														"command": "language-julia.chooseModule"
+													},
+													{
+														"key": "p",
+														"name": "Open package directory",
+														"icon": "new-folder",
+														"type": "command",
+														"command": "language-julia.openPackageDirectory"
+													},
+													{
+														"key": "t",
+														"name": "Tag new package version (experimental)",
```

Comment:
I am a little worried of putting `(experimental)` in the name. Once the underlying julia extension updated (so the tag package version is no longer experimental), the bindings here are likely not updated unless another PR.

I would say either remove this binding, or remove the word "experimental".

Reply:
Removed "experimental"...